nifi-extension-bundles/nifi-aws-bundle/nifi-aws-parameter-providers/src/main/java/org/apache/nifi/parameter/aws/AwsSecretsManagerParameterProvider.java

```diff
@@ -68,16 +72,73 @@
 @CapabilityDescription("Fetches parameters from AWS SecretsManager.  Each secret becomes a Parameter group, which can map to a Parameter Context, with " +
         "key/value pairs in the secret mapping to Parameters in the group.")
 public class AwsSecretsManagerParameterProvider extends AbstractParameterProvider implements VerifiableParameterProvider {
+    enum ListingStrategy implements DescribedValue {
+        Enumeration(
+                "Enumeration",
+                "Enumerate secret names",
```

Comment:
It is not necessary to have a `value` member, instead the `name()` of the enum can be used. Display names should use Title Case.
```suggestion
                "Enumerate Secret Names",
```